cpp/src/parquet/file_writer.cc

```diff
@@ -360,6 +384,10 @@ class FileSerializer : public ParquetFileWriter::Contents {
       }
       row_group_writer_.reset();
 
+      // In Parquet standard, the Bloom filter data can be stored before the page indexes
+      // after all row groups or stored between row groups. We choose to store it before
+      // the page indexes after all row groups.
```

Comment:
We can also say parquet-mr does the same thing.

Reply:
Ok, but give a rationale for this choice?

Reply:
> Ok, but give a rationale for this choice?

I'll implement this version first, and implement the choice or config later.

Reply:
Putting all bloom filters together may provide a good chance to coalesce I/Os of different bloom filters. Especially when only one column has enabled it, which is the common case.

Reply:
Yeah currently it's easy to implement like this. We can support other options in the future.